### ISSUE TITLE
Issue with aria-label selection for c-price--offer

### ISSUE THREAD

**Issue by sbradshaw** [sky-uk/toolkit#465]
**Describe the bug**
A position: relative is missing from c-price--offer which is causing a screen reader to skip over a tag with this class that has an aria-label set. The c-price--strike selector which is similar does have position: relative set and is working with screen readers.

**To Reproduce**
Using a standalone tag without any inherited CSS classes and and c-price--offer with an aria-label attribute, use a tool like VoiceOver to select the tag. It should be skipped unless position: relative is set.

**Expected behavior**
Should select a tag with aria-label set when navigation the page with a screen reader.

**Attempted Fixes**
adding position: relative to c-price--offer

**Screenshots**

**Desktop (please complete the following information):**

- OS: MacOS
- Browser chrome
- Version latest

**Smartphone (please complete the following information):**

**Additional context**
